dsgrid/config/time_dimension_base_config.py

```diff
@@ -181,10 +181,12 @@ def list_expected_dataset_timestamps(self, model_years=None):
 
         """
 
-    def _convert_time_to_project_time_interval(self, df, project_time_dim=None, wrap_time=True):
+    def _convert_time_to_project_time_interval(self, df, project_time_dim=None):
         """Shift time to match project time based on TimeIntervalType
-        If time range spills over into another year after time interval alignment,
+        If wrap_time_allowed in project_time_dim:
+        - If time range spills over into another year after time interval alignment,
         the time range will be wrapped around so it's bounded within the year
+        - Time will also wrapped if dataset has a different time zone than project
```

Comment:
Does this mean that the exact same function can be used to handle period-ending/period-beginning and different base time zones? (Neat!)